### DISCUSSION TITLE
adding support for a yespower blockchain

### DISCUSSION THREAD

**Discussion by xyephy** [trustwallet/wallet-core#3731] [Q&A]
Hello,
I want to add a new blockchain which is a bitcoin fork, the blockchain uses yespower for it mining algorithm. How do I make sure it seamlessly integrates with wallet core and does it support yespower functions?
Lastly how do I get the Coin ID or Value? (i.e for Litecoin I see TWCoinTypeLitecoin = 2)

**Comment by satoshiotomakan** [comment]
Hi @xyephy, please take a look at the guide: https://developer.trustwallet.com/developer/wallet-core/newblockchain

**Reply by xyephy** [reply]
@satoshiotomakan I went through the documentation I was just curious what if the coinID ("m/0'/0'/0'") is similar to Bitcoin how do I go about it in registry.json?

**Reply by xyephy** [reply]
any tips on how to go about this?

**Reply by satoshiotomakan** [reply]
There is a tip at the https://developer.trustwallet.com/developer/wallet-core/newblockchain/newevmchain page
> coinId: If own coinID is available, use that. Otherwise, use: 10000000 + chainID, such as 10008217.

**Reply by satoshiotomakan** [reply]
For example, you can take a look at the Cosmos chains

**Reply by xyephy** [reply]
Let me have a look at it, thanks
